coins/ethereum/src/error.rs

```diff
@@ -33,18 +33,15 @@ pub enum Error {
     /// Converted ParseInt error
     #[error("ParseInt error: {0}")]
     ParseInt(#[from] std::num::ParseIntError),
-    // Convert web3 error
-    // #[error("web3 error: {0}")]
-    // FromWeb3(#[from] web3::Error),
     /// Error related to converting from or to a hex
     #[error("Hex error: {0}")]
     Hex(#[from] hex::FromHexError),
     /// Error from the walletd_hd_key crate
     #[error("Error from walletd_hd_key: {0}")]
     WalletdHDKey(#[from] walletd_hd_key::Error),
-    /// Error when trying to initialize EthClient
-    #[error("Failed to initialize EthClient")]
-    EthClientInit,
+    /// Error when trying to initialize Solana client
+    #[error("Failed to initialize Solana client")]
+    SolanaClientInit,
```

Comment:
this isn't right the Ethereal module shouldn't be throwing Solana errors

Reply:
My bad. Silly on my side, I was using the Eth module as an example rather than the other way around